src/routes/docs/advanced/platform/roles/+page.markdoc

```diff
@@ -0,0 +1,287 @@
+---
+layout: article
+title: Roles
```

Comment:
Should we name this "Console roles" to help differentiate?

Reply:
How about Organization Roles?